recipe/build.sh

```diff
@@ -90,8 +90,7 @@ cmake -LAH -G "Ninja"                                                     \
     -DENABLE_CONFIG_VERIFICATION=ON                                       \
     $CPU_DISPATCH_FLAGS                                                   \
     $OPENMP                                                               \
-    -DLAPACK_LIBRARIES="-llapacke -llapack -lcblas -lblas"                \
-    -DWITH_LAPACK=1                                                       \
```

Comment:
I think the intent was to compile with lapacke by @isuruf 

That said, if it was "always broken" I'm ok Merging this so we can move forward with the other stuff.